tools/mdm/windows/poc-mdm-server/README.md

```diff
@@ -23,14 +23,27 @@ This code is MIT licensed and it was forked from [here](https://github.com/oscar
 ## Usage
 
 On the server side, you just need to run the project using the already provided cert and keys. The certificate is in `.pfx` file format, so you need to extract the certificate and key first, see https://stackoverflow.com/a/59120388/1094941.
+The "Import password" is "testpassword", and the names of the output files matter, on Linux something like this works (assuming you are in the certs/ directory):
```

Comment:
Just in case this raises eyebrows - the password was already in the doc (line 50 below), just making it clearer for that step.